x-pack/plugins/security_solution/public/detections/configurations/security_solution_detections/render_cell_value.tsx

```diff
@@ -37,8 +40,8 @@ export const RenderCellValue: React.FC<EuiDataGridCellValueElementProps & CellVa
 
   return (
     <GuidedOnboardingTourStep
-      isTourAnchor={isTourAnchor}
-      step={1}
+      isTourAnchor={isTourAnchor && !props.isDetails}
```

Comment:
Prevents this:
<img width="1596" alt="Screen Shot 2022-11-01 at 11 28 05 AM" src="https://user-images.githubusercontent.com/6935300/199587900-bd48ae28-a60d-4799-9f84-22050d1128ab.png">

Reply:
Is there any reason for the `!props.isDetails` to be outside the `const isTourAnchor = useMemo(...`?